module/cli_controller.py

```diff
@@ -4,95 +4,66 @@
 import common.frames
 from client.comm import BaseComm
 
-from module.command_node import NodeType, Node
+from module.command_node import GlobalCommand
 from module.command_tree_generator import load_commands
 from module.input_handler import input_handler
 
 
 class CLIController:
-    def __init__(self, comm: BaseComm, command_file_list=None):
+    def __init__(self, comm: BaseComm, command_file_list: list):
         """
         Filenames given have to be JSON files.
```

Comment:
I see that this code documentation was already existing. But I would suggest adding @param for explaining what comm and command_file_list are.